proto/tasks.proto

```diff
@@ -127,4 +127,7 @@ message Stats {
     //
     // If this is `None`, the task has not yet been woken.
     optional google.protobuf.Timestamp last_wake = 10;
+
+    // HdrHistogram.rs `Histogram` serialized to binary in the V2 + DEFLATE format
+    optional bytes poll_times_histogram = 11;
```

Comment:
adding the poll time histogram to `every` task stats update we send will probably increase the wire size of the update messages significantly --- although `hdrhistogram` is fairly compact, this is still potentially a lot of data.

here's an alternative proposal: when viewing a list of all tasks, i don't think we're going to actually care about the entire histogram. i think that we'll mostly want the histograms when we are displaying more detailed stats about individual task(s). one idea i've been considering is adding a new RPC to the [`Tasks` service](https://github.com/tokio-rs/console/blob/4ba86c027294503b3444ccd725bcc7a88562ba50/proto/tasks.proto#L9-L11) in the protobuf for requesting detailed data on an individual task. maybe something like this:
```protobuf
service Tasks {
    rpc WatchTasks(TasksRequest) returns (stream TaskUpdate) {}
    rpc TaskDetails(DetailsRequest) returns (stream TaskDetails) {}
}

// ...

message DetailsRequest {
    Task id = 1;
}

message TaskDetails {
    // HdrHistogram.rs `Histogram` serialized to binary in the V2 + DEFLATE format
    bytes poll_times_histogram = 1;
}

// ...
```
this way, we could always calculate histograms for each task in the aggregator (so that we can have historical data) but only serialize them on request. 

if we want to include some data from the histogram for all tasks in the `WatchTasks` RPC, we could add a couple percentile values to the `TasksUpdate` message (maybe the p50, p90, and p99 poll times?). those would be useful to display in overview views like the task list, and then when we want to show details about a specific task, we could start requesting its complete histogram.

what do you think? this would be a larger change, but it might be more efficient in terms of data on the wire. we could probably avoid paying the performance penalty from compressing the histograms in that case, because we wouldn't be sending the complete histogram for most tasks.

Reply:
This is a great idea, I was thinking that it could be made more efficient when playing with it on the TaskView side and trying to determine when to deserialize.  

I'm okay with it being a larger change. Though I jinxed myself when I said in the issue that I had lots of free time, I should be able to tackle this this week :)

Reply:
okay, that sounds great, i think this would be the right approach, especially when we add other detailed stats. let me know if you have any questions?